compiler/tests-common/tests/org/jetbrains/kotlin/codegen/debugInformation/AbstractDebugTest.kt

```diff
@@ -178,7 +181,7 @@ abstract class AbstractDebugTest : CodegenTestCase() {
         var inBoxMethod = false
         vmLoop@
         while (true) {
-            val eventSet = virtualMachine.eventQueue().remove(1000)
+            val eventSet = virtualMachine.eventQueue().remove(1000) ?: continue
```

Comment:
Seems it's equals to remove without timeout

Reply:
The difference is that, if there is no timeout I am not sure what will happen if it is timed out at system level. I don't know how to mock such behavior though, maybe it is already covered in the underlying implementation, but that's up to the JDK.